openrouteservice-api-tests/src/test/java/org/heigit/ors/v2/services/routing/ResultTest.java

```diff
@@ -2785,28 +2785,33 @@ public void testAlternativeRoutes() {
         coord2.put(8.689499);
         coord2.put(49.398295);
         coordinates.put(coord2);
+
         body.put("coordinates", coordinates);
         body.put("preference", "fastest");
         JSONObject ar = new JSONObject();
         ar.put("target_count", "2");
         ar.put("share_factor", "0.5");
         body.put("alternative_routes", ar);
-
+        body.put("extra_info", getParameter("extra_info"));
         given()
             .header("Accept", "application/json")
             .header("Content-Type", "application/json")
             .pathParam("profile", getParameter("carProfile"))
             .body(body.toString())
             .when()
             .post(getEndPointPath() + "/{profile}")
-            .then()
+            .then().log().all()
```

Comment:
again, better to just log if the validation fails

Reply:
ditto